src/components/DateRangePickerInput.jsx

```diff
@@ -123,6 +128,22 @@ export default class DateRangePickerInput extends React.Component {
     });
   }
 
+  renderInputIcon(calendarIcon) {
```

Comment:
Agreed: `renderFoo` methods are generally an antipattern; could this be an SFC, or inline jsx, instead?